server/services/service.ts

```diff
@@ -0,0 +1,10 @@
+import type { Request } from 'express'
```

Comment:
meh.. generally lower levels of your application shouldn't know about details of higher levels.

The service shouldn't necessarily know that it is living in an express app as dependencies only flow in one direction.

Reply:
Yeah, I get where you're coming from with this, it seemed a bit smelly when I was writing it, but I am very much a fan of DRY code, but typing `req.locals.token` everywhere is probably preferable to including external dependencies.